public/specifications/monitoring/MONITORING.md

```diff
@@ -0,0 +1,343 @@
+# Monitoring Specification
+
+---
+**Version:** 1.0 |
+**Date:** August 19, 2024
+
+JSON Schemas:
+[start-schema.json](/public/specifications/monitoring/schemas/start-schema.json ':ignore'),
+[error-schema.json](/public/specifications/monitoring/schemas/error-schema.json ':ignore'),
+[status-schema.json](/public/specifications/monitoring/schemas/status-schema.json ':ignore')
```

Comment:
Links are not working in the deployed preview?

Reply:
I would also suggest we move each scheme link to the corresponding section so that specs are closer to relevant information.

Reply:
JSON schema links have been moved closer to their definition. However, I leave this conversation open because of the commend below.

> Links are not working in the deployed preview?

I guess we'll find as soon as it's deployed. But in this case we'll have to choose which one should be accessible: the repo or the preview.

Reply:
> Links are not working in the deployed preview?

Since docsify deploys the content of `public`, I assume that the `/public` prefix on these URLs is too much.

Reply:
@MGaetan89 

> I guess we'll find as soon as it's deployed. But in this case we'll have to choose which one should be accessible: the repo or the preview.

But I agree with you, the `/public` breaks docsify page. I fixed the links. But it breaks links when viewed from the repo, if that matters.

Reply:
My fix is no bueno. I'll try something else tomorrow, I've some animes to watch.

Reply:
In my opinion, it's fine, since this file is in the `public` folder, which is meant to be used by docsify, more than by GitHub itself.

Reply:
Let's try merging and fix afterwards if this still does not work.